#==================================================================================================
# Temporary patch for build failures on 10.13/10.14; will be superseded by upstream's official
# fix, expected in the next release.
#
# Date:   2025-06-05
# Author: Tomio Arisaka / MacPorts
#
# See: https://trac.macports.org/ticket/71575#comment:19
# See: https://github.com/harfbuzz/harfbuzz/issues/5354
#==================================================================================================
--- src/hb-coretext.cc.orig	2025-05-13 01:51:07.000000000 +0900
+++ src/hb-coretext.cc	2025-06-05 17:44:30.000000000 +0900
@@ -207,7 +207,7 @@
   {
     // https://github.com/harfbuzz/harfbuzz/issues/5300
 #if (defined(__IPHONE_OS_VERSION_MIN_REQUIRED) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 110000) || \
-    (defined(__MAC_OS_X_VERSION_MIN_REQUIRED) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 101300) || \
+    (defined(__MAC_OS_X_VERSION_MIN_REQUIRED) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 101500) || \
     (defined(__TV_OS_VERSION_MIN_REQUIRED) && __TV_OS_VERSION_MIN_REQUIRED >= 110000) || \
     (defined(__WATCH_OS_VERSION_MIN_REQUIRED) && __WATCH_OS_VERSION_MIN_REQUIRED >= 40000) || \
     (defined(__MACCATALYST_VERSION_MIN_REQUIRED) && __MACCATALYST_VERSION_MIN_REQUIRED >= 130100) || \
